src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.ts

```diff
@@ -32,16 +32,35 @@ export const useTradeRoutes = (
     if (isEmpty(assets) || !feeAsset) return
 
     try {
-      const [sellAssetId, buyAssetId] = getDefaultPair()
-      const sellAsset = assets[sellAssetId]
+      const [defaultSellAssetId, defaultBuyAssetId] = getDefaultPair()
+      const sellAsset = assets[defaultSellAssetId]
 
-      // ugly hack until we add proper error handling in another PR soon
-      const buyAsset =
-        assets[
-          defaultBuyAssetId?.startsWith('eip155:1') && defaultBuyAssetId !== 'eip155:1/slip44:60'
-            ? defaultBuyAssetId
-            : buyAssetId
-        ]
+      const preBuyAssetToCheckId = routeBuyAssetId ?? defaultBuyAssetId
```

Comment:
This name confuses me.

Reply:
Had to define this variable to do the logic on line 41 without using the let pattern